README.markdown

```diff
@@ -179,57 +225,84 @@ Tells Puppet whether the key should or shouldn't exist. Valid options: 'present'
 
 Specifies a Registry key for Puppet to manage. If any of the parent keys in the path don't exist, Puppet creates them automatically. Valid options: a string containing a Registry path. For example: `HKLM\Software` or `HKEY_LOCAL_MACHINE\Software\Vendor`.
 
-If Puppet is running on a 64-bit system, the 32-bit Registry key can be explicitly managed using a prefix. For example: `32:HKLM\Software`.
+If Puppet is running on a 64-bit system, manage the 32-bit Registry key using a prefix, for example: `32:HKLM\Software`.
 
 ##### `purge_values`
 
 *Optional.* Specifies whether to delete any values in the specified key that are not managed by Puppet. Valid options: `true` and `false`. Default value: `false`.
 
 For more on this parameter, see the [Purge existing values section](#purge-existing-values) under Usage.
 
+
 #### `registry_value`
 
-##### `path`
+Manages individual Registry values.
 
-*Optional.* Specifies a Registry value for Puppet to manage. Valid options: a string containing a Registry path. If any of the parent keys in the path don't exist, Puppet creates them automatically. For example: `HKLM\Software` or `HKEY_LOCAL_MACHINE\Software\Vendor`. Default value: the title of your declared resource.
+The title of the resource determines the path and value name that will be managed, for example:
 
-If Puppet is running on a 64-bit system, the 32-bit Registry key can be explicitly managed using a prefix. For example: `32:HKLM\Software\Value3`.
+The manifest below manages the value called 'foo' in the key 'HKLM\Software'
 
-##### `ensure`
+``` puppet
+registry_value { 'HKLM\Software\foo':
+  ...
+}
+```
 
-Tells Puppet whether the value should or shouldn't exist. Valid options: 'present' and 'absent'. Default value: 'present'.
+You can also manage the default value of a key by adding a backslash at the end of the path.  The manifest below manages the default value in the key 'HKLM\Software\Vendor'
 
-##### `type`
+``` puppet
+registry_value { 'HKLM\Software\Vendor\\':
+  ...
+}
+```
 
-*Optional.* Sets the data type of the specified value. Valid options: 'string', 'array', 'dword', 'qword', 'binary' and 'expand'. Default value: 'string'.
+Note you can explicitly set the path and value using the `path` or `value_name` parameters. These are used instead of the resource title.  When explicitly setting a `path` or `value_name` parameter, use both to avoid unexpected behavior, for example:
 
-##### `data`
+The manifest below manages the value called 'value' (extracted from the title because `value_name` is not specified) in the key 'HKLM\Software\foo'
```

Comment:
Nit - I think you want `HKLM\Software` instead of 'HKLM\Software' to be consistent with rest of README